src/rust/ide/src/view/project_name.rs

```diff
@@ -0,0 +1,102 @@
+//! This module provides a view for project's name which can be used to edit it.
```

Comment:
This project name view is wrong. It contains controller in view. Tehre is another PR which implements that in a correct way. This should not be part of this PR. I was writing about it earlier. This code bundles view and controller and should not land on master.

Reply:
@wdanilo this file is completely removed in https://github.com/enso-org/ide/pull/639. My suggestion is:

1. Once https://github.com/enso-org/ide/pull/639 is approved, I can merge it into this PR's branch.
2. Then merge everything into `main`.

Reply:
Lets do it this way :)